5001-6000/LIT5696MagicPrayer.xml

```diff
@@ -37,7 +37,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p><foreign xml:lang="gez">Ḥaṣura masqal</foreign> is a generic title for protective prayers that
```

Comment:
mark up `<ref type="authFile" target="TrueCross">Holy Cross</ref>` and/or add relation
`<relation name="ecrm:P129_is_about" active="LIT5696MagicPrayer" passive="TrueCross">`

Reply:
done